content/authors/guoxuchen/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+# Display name
+title: Guoxu Chen
+
+# Username (this should match the folder name)
+authors:
+- guoxuchen
```

Comment:
The `username` should match the folder name. If the folder is guoxuchen, the name should be `Guoxu Chen` instead of guoxuchen. Same for other authors. See https://github.com/MIGG-NTU/academic/edit/main/content/authors/jiayuanyao/_index.md for an example.
```suggestion
- Guoxu Chen
```